pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -2502,23 +2502,27 @@ var _ = Describe("getDetachedDisks", func() {
 			[]api.Disk{}),
 	)
 })
-
 var _ = Describe("migratableDomXML", func() {
 	var ctrl *gomock.Controller
 	var mockDomain *cli.MockVirDomain
 	BeforeEach(func() {
 		ctrl = gomock.NewController(GinkgoT())
 		mockDomain = cli.NewMockVirDomain(ctrl)
 	})
-	It("should remove metadata", func() {
+	It("should parse the XML with the metadata", func() {
 		domXML := `<domain type="kvm" id="1">
   <name>kubevirt</name>
-  <kubevirt><migration>this should stay</migration></kubevirt>
+  <metadata>
+    <kubevirt xmlns="http://kubevirt.io">
+    </kubevirt>
+   </metadata>
 </domain>`
-		// migratableDomXML() removes the migration block but not its ident, which is its own token, hence the blank line below
 		expectedXML := `<domain type="kvm" id="1">
   <name>kubevirt</name>
-  <kubevirt><migration>this should stay</migration></kubevirt>
```

Comment:
Yes, this was wrong before. Using libvirtxml exposed it and it is nice that it works (custom Metadata struct inside libvirt domain)